tiny_dnn/core/kernels/tiny_quantized_deconv2d_kernel.h

```diff
@@ -279,13 +279,6 @@ inline void tiny_quantized_deconv2d_back_kernel(const deconv_params &params,
     &min_prev_delta_requantized, &max_prev_delta_requantized,
     &prev_delta_requantized);
 
-  // dequantize to flaot, this could be removed within concatenated quantized
-  // network
-  vec_t prev_delta_vec = quantized_tensor_to_float<uint8_t>(
-    prev_delta_requantized, min_prev_delta_requantized,
-    max_prev_delta_requantized);
-  prev_delta = &prev_delta_vec;
-
   // Accumulate dw
   for_i(params.in.depth_, [&](int inc) {
     for (serial_size_t outc = 0; outc < params.out.depth_; outc++) {
```

Comment:
Unbalanced braces. I ahve a feeling that you just mistakenly removed random part of the code